src/libraries/System.Security.Cryptography.X509Certificates/tests/ChainTests.cs

```diff
@@ -175,7 +181,10 @@ public static void TestResetMethod()
                 Assert.False(valid);
                 chainHolder.DisposeChainElements();
 
+                // Make sure AllowUnknownCertificateAuthority is set, as this test we verifies it does not get reset
```

Comment:
Can you reword this?

Reply:
Just throwing random words together over here...